### ISSUE TITLE
Detection of FPs

### ISSUE THREAD

**Issue by Ivan1905** [sprtd/rake-scam-token-forta-detection-bot#7]
Hi sprtd,

Adding a few checker that could make this bot a little more precise.

1. Considering the bot is labeling addresses which token may have been deployed a while ago. The bot could check that if there are still trades on a token which was deployed more than x days ago, then there are less chances that the token is rake. For instance [this](https://explorer.forta.network/alert/0xb0b12cbd7ae48d1f7b3fe12f2955912e088d4f3baac781c4ff21306c6e8d7886) alert labeled [this](0x99B600D0a4abdbc4a6796225a160bCf3D5Ce2a89) token as rake and it shouldn't be (considering the fees a that are taken are specified in their [tokenomics](https://solareum.gitbook.io/solareum-a-peer-to-peer-renewable-energy-system/tokenomics#auto-mining-consensus-protocol-and-distribution))
2. Also it may be the case that there are fees charged in the token that was created and then those fees burn. Maybe there could be a checker that if deployer interacts with 0x000000000000000000000000000000000000dEaD (burn address) there are less chances the token is rake.

**Comment by sprtd** [comment]
@Ivan1905 For [2](https://github.com/sprtd/rake-scam-token-forta-detection-bot/issues/7#issue-1897000568),  please can you add an instance of such triggered alert if you have any

**Comment by Ivan1905** [comment]
yes, sharing a tx https://explorer.phalcon.xyz/tx/eth/0x5e533573b08614ce949178d4fff284630670c43d6bd87b5a70ff88dd821c6ea1

please let me know if I'm looking things the wrong way.
